packages/x-date-pickers/src/DateTimePicker/DateTimePickerTabs.tsx

```diff
@@ -101,9 +93,7 @@ const DateTimePickerTabs = function DateTimePickerTabs(inProps: DateTimePickerTa
   } = props;
 
   const localeText = useLocaleText();
-  const wrapperVariant = React.useContext(WrapperVariantContext);
-  const ownerState = { ...props, wrapperVariant };
-  const classes = useUtilityClasses(ownerState);
+  const classes = useUtilityClasses(props);
```

Comment:
Most of the time, we keep the `const ownerState = props` 

I don't have a strong opinion on this though

Reply:
66 vs 13 instances, but also most of the time where `ownerState` is passed—it is an extension of `props` or even a custom object built from other props.
IMHO, I see little to no sense in creating a temporary variable reference copy just to pass the same data to a function. 🤔 
In cases where we reuse the `ownerState` for other reasons (need to pass `ownerState` somewhere else) or augment the `props` object with extra info, then I do totally agree with using `ownerState`. 👌